test/infrastructure/docker/templates/cluster-template-development.yaml

```
@@ -24,7 +24,7 @@ spec:
       value: ""
     - name: podSecurityStandard
       value:
-        enabled: ${POD_SECURITY_STANDARD_ENABLED:=true}
+        enabled: ${ENABLE_POD_SECURITY_STANDARD:=true}
```

This now swaps the issue around.

If you change the doc to `POD_SECURITY_STANDARD_ENABLED`, then this should still be `POD_SECURITY_STANDARD_ENABLED`

```suggestion
        enabled: ${POD_SECURITY_STANDARD_ENABLED:=true}
```

---

(kudos to @fabriziopandini )

---

Let's use this one `POD_SECURITY_STANDARD_ENABLED` everywhere